CHANGELOG.md

```diff
@@ -19,6 +20,7 @@
   * Remove `PaymentMethodDeleteException`
   * Remove `GetPaymentMethodNoncesCallback`
   * Remove `DeletePaymentMethodNonceCallback`
+  * Update `GooglePayIsReadyToPayCallback` to expect a boolean primitive instead of a Boxed type
```

Comment:
I noticed that there's a few other places that we've updated to use primitives instead of boxed types. Any reason we're calling out this one and not the others?